tests/python/pants_test/backend/python/tasks/test_python_distribution_integration.py

```diff
@@ -190,6 +190,29 @@ def test_pants_tests_local_dists_for_current_platform_only(self):
       pants_run = self.run_pants(command=command, config=pants_ini_config)
       self.assert_success(pants_run)
 
+  def test_pants_native_source_detection_when_running_local_dists_for_current_platform_only(self):
+    # Test that `./pants run` respects python_binary platforms
+    # arguments when the closure contains native sources. To do this,
+    # we need to setup a pants.ini that contains two platform defauts:
+    # (1) "current" and (2) a different platform than the one we are currently
+    # running on. The test target below has `platforms=("current")`.
+    non_current_platform = None
+    platform_string = Platform.create().normalized_os_name
+    if platform_string == 'darwin':
+      non_current_platform_string = 'linux-x86_64'
+    else:
+      non_current_platform_string = 'macosx-10.12-x86_64-x86_64'
```

Comment:
Can be simplified to 

```python
non_current_platform_string = 'macosx-10.12-x86_64-x86_64' if platform_string == 'darwin' else 'linux-x86_64'
```

I'm not sure what the style guide / project convention prefers, though. Some people find the original as better.

Reply:
Sure thing